examples/multilang/nodejs/index.ts

```diff
@@ -0,0 +1,18 @@
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+import * as mycomponent from "../mycomponent/nodejs";
+
+// This should go inside `@pulumi/aws`.
+pulumi.runtime.registerProxyConstructor("aws:ec2/securityGroup:SecurityGroup", aws.ec2.SecurityGroup);
+
+////////////////////////////////
+// This is code the user would write to use `mycomponent` from the guest language.
+
+const res = new mycomponent.MyComponent("n", {
+    input1: Promise.resolve(42),
+}, { ignoreChanges: ["input1"] /*, providers: { "aws": awsProvider } */ });
```

Comment:
why the ignorechanges?

Reply:
An attempt to test resource options.  I'm not sure this is actually working correctly yet though.